cmd/kubeadm/app/util/runtime/runtime.go

```diff
@@ -140,10 +140,15 @@ func (runtime *CRIRuntime) RemoveContainers(containers []string) error {
 func (runtime *DockerRuntime) RemoveContainers(containers []string) error {
 	errs := []error{}
 	for _, container := range containers {
-		out, err := runtime.exec.Command("docker", "rm", "--force", "--volumes", container).CombinedOutput()
+		out, err := runtime.exec.Command("docker", "stop", container).CombinedOutput()
```

Comment:
AFAIK this can block for 10 seconds for a container (default -t)
does that mean that if a lot of containers are running the overall process will take a lot of time?

do we need additional output messages indicating what is going on here?